shell/platform/darwin/macos/framework/Source/FlutterIntermediateKeyResponder.h

```diff
@@ -0,0 +1,24 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import <Cocoa/Cocoa.h>
+
+/*
+ * An interface for adding key responders that can declare itself as the final
+ * responder of the event, terminating the event propagation.
+ *
+ * It differs from an NSResponder in that it returns a boolean from the
+ * handleKeyUp and handleKeyDown calls, where true means it has handled the
+ * given event. A handled event is not passed to the next responder.
```

Comment:
This last sentence isn't part of the responsibilty of this class; saying that here makes it sound like this class passes unhandled events, which isn't true.

Reply:
OK, I just removed the last sentence.